spec/models/notification_service/slack_service_spec.rb

```diff
@@ -9,14 +9,18 @@
                                            service_url: service_url
   end
 
-  it "it should send a notification to Slack with hook url" do
+  it "should have icon for slack" do
+    expect(Rails.root.join("docs/notifications/slack/errbit.png")).to exist
+  end
+
+  it "should send a notification to Slack with hook url" do
     # setup
     problem = notice.problem
 
     # faraday stubbing
     payload = {
       username:    "Errbit",
-      icon_emoji:  ":collision:",
+      icon_url:    "https://raw.githubusercontent.com/errbit/errbit/master/docs/notifications/slack/errbit.png",
```

Comment:
Is this normal? It seems a little strange to load the icon through raw.githubusercontent.com. Wouldn't it be more normal to put the icon in the errbit application and let errbit host it?